app/src/redux-resources/analytics/hooks/useCameraAnalytics.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  ANALYTICS_CAMERA_ENABLEMENT_KIND,
+  ANALYTICS_CAMERA_SETTINGS_KIND,
+  ANALYTICS_IMAGE_CAPTURE_KIND,
+  ANALYTICS_LIVE_FEED_KIND,
+  ANALYTICS_PHOTO_ACCESS,
+  useTrackEvent,
+} from '/app/redux/analytics'
+
+import type { RobotType } from '@opentrons/shared-data'
+
+export const SOURCE_RUN_RECORD = 'runRecord'
+export const SOURCE_ROBOT_SETTINGS = 'robotSettings'
+
+export interface CameraAnalyticsParams {
+  source: typeof SOURCE_RUN_RECORD | typeof SOURCE_ROBOT_SETTINGS
+  robotType: RobotType
+}
+
+export interface CameraUsageParams {
+  cameraEnabled: boolean
+  liveFeedEnabled: boolean
+  recoveryCaptureEnabled: boolean
+}
+
+interface CameraSettingsParams {
+  settingsType: 'zoom' | 'saturation' | 'brightness' | 'contrast'
+  value: number | string
+}
+
+interface CaptureParams {
+  amount: number
+  runId: string
+  errorDetails?: string
+}
+
+interface PhotoAccessParams {
+  action: 'download' | 'downloadZip' | 'delete' | 'storageWarning'
+  transactionId?: string
+}
+
+export interface UseCameraUsageAnalyticsResult {
+  /* Reports when camera enablement settings change. */
+  reportCameraEnablementSettings: (data: CameraUsageParams) => void
+  /* Reports when camera view settings change. */
+  reportCameraSettings: (data: CameraSettingsParams) => void
+  /* Reports image capture rate and sources. */
+  reportImageCaptureUsage: (data: CaptureParams) => void
+  /* Reports live feed usage. */
+  reportLiveFeedUsage: (data: CaptureParams) => void
+  /* Reports how often images are downloaded together, seperately, 
+  how often images are deleted, and how often storage warnings appear. */
+  reportPhotoAccessUsage: (data: PhotoAccessParams) => void
+}
+
+export function useCameraAnalytics({
+  robotType,
+  source,
+}: CameraAnalyticsParams): UseCameraUsageAnalyticsResult {
+  const doTrackEvent = useTrackEvent()
+
+  const reportCameraEnablementSettings = (data: CameraUsageParams): void => {
+    doTrackEvent({
+      name: ANALYTICS_CAMERA_ENABLEMENT_KIND,
+      properties: {
+        robotType,
+        source,
+        cameraEnabled: data.cameraEnabled,
+        liveFeedEnabled: data.liveFeedEnabled,
+        recoveryCaptureEnabled: data.recoveryCaptureEnabled,
+      },
+    })
+  }
+  const reportCameraSettings = (data: CameraSettingsParams): void => {
+    doTrackEvent({
+      name: ANALYTICS_CAMERA_SETTINGS_KIND,
+      properties: {
+        robotType,
+        source,
+        value: data.value,
+        settingsType: data.settingsType,
+      },
+    })
+  }
+
+  const reportImageCaptureUsage = (data: CaptureParams): void => {
+    doTrackEvent({
+      name: ANALYTICS_IMAGE_CAPTURE_KIND,
+      properties: {
+        robotType,
+        source,
+        runId: data.runId,
```

Comment:
```suggestion
        transactionId: data.runId,
```
`transactionId` is the keyword Product knows that if present, they need to filter by uniqueness.